test/e2e/utils/constants.go

```diff
@@ -23,6 +23,8 @@ var (
 
 	LogOutputDir = constants.GetEnvWithDefault("TESTING_LOG_DIR", os.TempDir()+"/infinispan-operator")
 
+	Debug, _ = strconv.ParseBool(constants.GetEnvWithDefault("DEBUG", "false"))
```

Comment:
This would be more flexible if we allow the log level to be specified via the env variable, something like:

```go
var (
...
        LogLevel     = logLevel()
	LogOutputDir = constants.GetEnvWithDefault("TESTING_LOG_DIR", os.TempDir()+"/infinispan-operator")
...
)

func logLevel() zapcore.Level {
	level, err := zapcore.ParseLevel(constants.GetEnvWithDefault("LOG_LEVEL", "info"))
	ExpectNoError(err)
	return level
}
```

Then we can replace `if Debug` with `if Log().Level().Enabled(zapcore.DebugLevel) {...}`

Reply:
That's basically what I had initially, then I came across `DEBUG` option in the `httpClient` and it go me thinking. At the moment ZAP doesn't include possibility to TRACE with debug being the lowest level, so you have by INFO default and you can go only for DEBUG. And I didn't see a use case for enabling only WARN levels and above either. (And running `DEBUG=true go test -v ...` was a bit faster compared to `TESTING_LOG_LEVEL=debug go test -v ...`)

Neverthless I'm pretty happy implement the suggestion as it's a valid solution and should ZAP introduce TRACE logging that it'd make even more sense